[2025-02-12T09:19:52.060+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-12T09:19:52.168+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: e_commerce_dw_etl.transform.transform_dim_sellers manual__2025-02-12T09:18:49.558799+00:00 [queued]>
[2025-02-12T09:19:52.197+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: e_commerce_dw_etl.transform.transform_dim_sellers manual__2025-02-12T09:18:49.558799+00:00 [queued]>
[2025-02-12T09:19:52.199+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-12T09:19:52.280+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform.transform_dim_sellers> on 2025-02-12 09:18:49.558799+00:00
[2025-02-12T09:19:52.297+0000] {standard_task_runner.py:63} INFO - Started process 360 to run task
[2025-02-12T09:19:52.307+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'e_commerce_dw_etl', 'transform.transform_dim_sellers', 'manual__2025-02-12T09:18:49.558799+00:00', '--job-id', '216', '--raw', '--subdir', 'DAGS_FOLDER/e_commerce_dw_dag.py', '--cfg-path', '/tmp/tmpb3_at552']
[2025-02-12T09:19:52.314+0000] {standard_task_runner.py:91} INFO - Job 216: Subtask transform.transform_dim_sellers
[2025-02-12T09:19:52.464+0000] {task_command.py:426} INFO - Running <TaskInstance: e_commerce_dw_etl.transform.transform_dim_sellers manual__2025-02-12T09:18:49.558799+00:00 [running]> on host 1f90b0c5673e
[2025-02-12T09:19:52.731+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='e_commerce_dw_etl' AIRFLOW_CTX_TASK_ID='transform.transform_dim_sellers' AIRFLOW_CTX_EXECUTION_DATE='2025-02-12T09:18:49.558799+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-12T09:18:49.558799+00:00'
[2025-02-12T09:19:52.733+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-12T09:19:52.816+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-02-12T09:19:53.087+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-02-12T09:19:53.305+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-02-12T09:19:53.567+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/providers/common/sql/hooks/sql.py:263 UserWarning: pandas only supports SQLAlchemy connectable (engine/connection) or database string URI or sqlite3 DBAPI2 connection. Other DBAPI2 objects are not tested. Please consider using SQLAlchemy.
[2025-02-12T09:19:54.053+0000] {base.py:84} INFO - Using connection ID 'postgres' for task execution.
[2025-02-12T09:19:55.470+0000] {logging_mixin.py:188} INFO - Đã transform và lưu dữ liệu vào dim_sellers
[2025-02-12T09:19:55.475+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-02-12T09:19:55.478+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-12T09:19:55.502+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=e_commerce_dw_etl, task_id=transform.transform_dim_sellers, run_id=manual__2025-02-12T09:18:49.558799+00:00, execution_date=20250212T091849, start_date=20250212T091952, end_date=20250212T091955
[2025-02-12T09:19:55.633+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-12T09:19:55.748+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T09:19:55.765+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
